src/components/EditPost/EditPost.jsx

```diff
@@ -81,68 +85,86 @@ const EditPost = ({
-          <h4>I would like to write a(n)</h4>
-          <select onChange={handlePostTypeSelect}>
-            <option>Update or Achievement</option>
-            <option>Event Announcement</option>
-            <option>Job Opportunity</option>
-            <option>Educational Content</option>
-            <option>Industry Insight</option>
-          </select>
-          <h4> post in a</h4>
-          <select onChange={handleMoodSelect}>
-            <option>Professional</option>
-            <option>Witty</option>
-            <option>Appreciative</option>
-            <option>Casual</option>
-            <option>Engaging</option>
-            <option>Motivating</option>
-          </select>
-          <h4>tone</h4>
-        </div>
-        <h4>What do you want to share?</h4>
-        <InputBox
-          value={inputText}
-          onChange={handleInputChange}
-          placeholder="Help me write a professional sounding post about my upcoming community event."
-          className="promptpage__input-box"
-          labelVal="Give AI specific instructions for your content"
-        />
-        <h4>Output size</h4>
-        {/* this will need to be within form, along with everything else */}
-        <div className="promptpage__choice-selection">
-          <input type="radio" id="small" name="output-size" value="small"></input>
-          <label htmlFor="small">Small</label><br></br>
-          <input type="radio" id="med" name="output-size" value="med"></input>
-          <label htmlFor="med">Medium</label><br></br>
-          <input type="radio" id="large" name="output-size" value="large"></input>
-          <label htmlFor="large">Large</label><br></br>
+    <div className="promptpage__container">
+      <div className="promptpage__input-container">
+        <PromptHeader headerText={"CommitAI Content Generator"} />
+        <div className="promptpage__sub-container">
+          <div className="promptpage__mood-box">
+            <h4>I would like to write a(n)</h4>
+            <select onChange={handlePostTypeSelect}>
+              <option>Update or Achievement</option>
+              <option>Event Announcement</option>
+              <option>Job Opportunity</option>
+              <option>Educational Content</option>
+              <option>Industry Insight</option>
+            </select>
+            <h4> post in a</h4>
+            <select onChange={handleMoodSelect}>
+              <option>Professional</option>
+              <option>Witty</option>
+              <option>Appreciative</option>
+              <option>Casual</option>
+              <option>Engaging</option>
+              <option>Motivating</option>
+            </select>
+            <h4>tone</h4>
+          </div>
+          <h4>What do you want to share?</h4>
+          <InputBox
+            value={inputText}
+            onChange={handleInputChange}
+            placeholder="Help me write a professional sounding post about my upcoming community event."
```

Comment:
I think we can change the placeholder text now that the post type & tone are in the dropdown

Reply:
Done! I've changed it to what the design team has in the mock up: "What is this about?"